_includes/wna-donate-form

```diff
@@ -0,0 +1,32 @@
+<div class="donation-form-container" style="text-align:left;">
```

Comment:
This file is unchanged from the donate form- will you be using a different form?

Reply:
For now, we will be using the default till we create new campaigns for the upcoming projects. Along the line when new campaigns are created, I will have to create new forms per request(I'd have to sync with you if that's the best way).